example/src/modules/x/hasMockedInternals/hasMockedInternals.spec.js

```diff
@@ -45,4 +44,17 @@ describe('<x-has-mocked-internals>', () => {
     // Make assertions about post-hydrated DOM.
     expect(querySelectorDeep('div#changeme')).to.have.text('new value');
   });
+
+  it('works succesfully with partial mocks', async () => {
+    await mockDep(`
+      export default 'bar';   // I can partially mock one export i.e default here,it does not affect the values of other unmocked exports
```

Comment:
```suggestion
      // I can partially mock one export i.e default here; it does not affect the
      // values of other unmocked exports
      export default 'bar';
```

Minor suggestion for readability.